modules/release-notes/con-relnotes-known-issues.adoc

```diff
@@ -22,3 +22,12 @@ If you deploy {product-short} without a proxy, there is the potential for IP add
 * Prevent collection of IP addresses by configuring a reverse proxy in front of {product-short}.
 //link:{LinkRHIDPIssue}RHIDP-2766[RHIDP-2766]
 --
+
+Rollbacks are not supported::
++
+--
+If you deploy {product-short} using either Helm or the Operator, bear in mind that rollbacks are not supported.
+It is strongly advised to take regular backups of your {product-short} PostgreSQL database, to be able to restore the application in case of issues.
```

Comment:
```suggestion
Performing regular backups of your {product-short} PostgreSQL database helps you to restore the application to a recent state in case of issues.
```

We avoid language such as "recommended" or "advised" or "strongly encouraged/discouraged". Instead, we simply state the recommended approach and the consequences of taking different approach.